Documentation/Signoffs/Extra.md

```diff
@@ -0,0 +1,81 @@
+# Extra
+
+![System](../Images/Extra_jrneal20.png)
+
+Figure 1: Extra subsystem and the pause switch can be seen in this figure.
+
+- The goal of this subsystem is to adhere to the rules as provided by the customer, Devcom, and to add lights and sounds to the design which will add to the point total as provided by Devcom. The system needs a pause switch to deactivate it between rounds. The system will also include lights and sounds before launching a projectile to add to the decoration portion of the design.
+
+## **Constraints:**
+
+| **Number:** | **Constraint:** | **Origin:** | 
+| --- | --- | --- |
+| 1. |  The interceptor shall have a switch that sets the system into a pause state that will keep the interceptor from firing. | Rulebook |
+| 2. | The voltage switched by the pause switch shall be 5V. | System Constraint|
+| 3. | The interceptor must have lights that are bright enough to be seen by the judges. These lights must be powered by a 5V source from the processor block and must emit a light that falls within the visual light spectrum of 380 to 720 nanometers [1]. | Rulebook / System |
+| 4. | The interceptor must make sounds before firing. The sounds will be powered by a voltage of 5V and the sounds will need to fall within the range of human hearing which is 20 to 20,000Hz or 0 - 120db [2]. The sound that is played should not damage anyone's ears therefore it should not be over 85db for a long period of time [2]. | Rulebook / Safety |
```

Comment:
db and hz are not interchangeable and do not convey similar information. This constraint needs to be reconsidered and background reading on what is conveyed by db and hz is needed.